src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/GetDescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,154 @@
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201 or 200
+
+        @ignore
+        Scenario: 01 Verify existing descriptors can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    [
+                        {
+                            "id": "{id}",
+                            "codeValue": "Sick Leave",
+                            "description": "Sick Leave",
+                            "effectiveBeginDate": "2024-05-14",
+                            "effectiveEndDate": "2024-05-14",
+                            "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                            "shortDescription": "Sick Leave"
+                        }
+                    ]
+                  """
+
+        @ignore
+        Scenario: 02 Verify retrieving a single descriptor by ID
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                      "id": "{id}",
+                      "codeValue": "Sick Leave",
+                      "description": "Sick Leave",
+                      "effectiveBeginDate": "2024-05-14",
+                      "effectiveEndDate": "2024-05-14",
+                      "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                      "shortDescription": "Sick Leave"
+                    }
+                  """
+
+        @ignore
+        Scenario: 03 Verify response code 404 when ID does not exist
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/123123123123"
+             Then it should respond with 404
+
+        @ignore
+        Scenario: 04 Verify array records content on GET All
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+              And total of records should be 1
+
+        @ignore
+        Scenario: 05 Ensure clients can create a Descriptor with the required attributes
+             When a POST reques is made to "/ed-fi/disabilityDescriptors" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Deaf-Blindness",
+                      "shortDescription": "Deaf-Blindness",
+                  }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: 06 Ensure clients can retrieve optional attributes from descriptors that have used them
```

Comment:
Scenario 1 already covers optional attributes.

Actually, the Given statement is ignoring the optional attributes, and that makes for a useful test. Please change the scenario "Read a descriptor that only contains required attributes".

Then you need to adjust the response body to remove the three optional attributes.

Reply:
Changed